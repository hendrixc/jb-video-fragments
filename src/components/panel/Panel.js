import React, { Component } from 'react';
import './panel.css';
import ClipList from '../clips/ClipList';

class Panel extends Component {
  render() {
    return (
      <div className="panel border-panel">
        <ClipList />
      </div>
    );
  }
}

export default Panel;
