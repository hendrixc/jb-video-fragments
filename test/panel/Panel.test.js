import React from 'react';
import renderer from 'react-test-renderer';
import Panel from '../../src/components/panel/Panel';

jest.mock('../../src/components/clips/ClipList', () => 'ClipList');

it('renders an home component into the Panel as snapshot', () => {
  const TextInputComponent = renderer.create(<Panel />).toJSON();
  expect(TextInputComponent).toMatchSnapshot();
});
