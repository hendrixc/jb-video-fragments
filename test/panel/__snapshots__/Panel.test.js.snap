// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders an home component into the Panel as snapshot 1`] = `
<div
  className="panel border-panel"
>
  <ClipList />
</div>
`;
